addons/AllSkyFree/AllSkyFree_OvercastLow.tscn

```diff
@@ -34,7 +34,7 @@ glow_bloom = 0.02
 glow_hdr_threshold = 0.44
 glow_bicubic_upscale = true
 
-[node name="AllSkyFree_Sky_OvercastLow" type="Node"]
+[node name="AllSkyFree_OvercastLow" type="Spatial"]
```

Comment:
Actually, the root node should be `Node`, because this scene is not intended to be moved around. When this node is `Node` instead of `Spatial`, Godot doesn't have to calculate the transforms.

Same with all the other scenes.